app/utils/createEntityReducer.js

```diff
@@ -119,7 +138,7 @@ export function optimistic(mutateType?: ?AsyncActionType) {
   };
 }
 
-export function paginationReducer(key: string, fetchType?: ?AsyncActionType) {
+export function paginationReducer(key: string, fetchType: AsyncActionType) {
```

Comment:
Maybe no need for a key here either? `insertEntities` uses the key in an assertion with the schema key, but here the pagination key is just retrieved.

Reply:
:+1: